crates/ibc/src/applications/transfer/denom.rs

```diff
@@ -153,6 +159,14 @@ impl PrefixedDenom {
     pub fn add_trace_prefix(&mut self, prefix: TracePrefix) {
         self.trace_path.add_prefix(prefix)
     }
+
+    pub fn trace_path(&self) -> &TracePath {
```

Comment:
It seems to me like we should simply make `trace_path` and `base_denom` public and remove all methods at this point. WDYT @hu55a1n1 ?

Reply:
@DaviRain-Su would you like to make this change and update the changelog?

Reply:
ok nice